src/subcommandlineparser.cpp

```diff
@@ -39,29 +39,27 @@ static const std::map<QString, QList<QCommandLineOption>> options{
           QObject::tr("NAME"),
       },
       {
-          {"q", "object"},
-          QObject::tr("The name of the object to render. "
-                      "Keep clear to render the whole scene. "
-                      "The name does not need to be unique. "
-                      "In this case, all matching objects are rendered. "
+          {"q", "object-name"},
+          QObject::tr("The name of the object to render."
```

Comment:
The space at the end of the strings was not a mistake.
If the strings are concatenated, the space is required, IIRC.

Reply:
Oh yes, didn't realize they're concatenated, thanks for pointing it out.